src/libData/AccountData/AccountStoreSC.cpp

```diff
@@ -1449,9 +1464,11 @@ bool AccountStoreSC<MAP>::ParseCallContractJsonOutput(
       }
 
       if (is_library) {
+        // Scilla should be invoked for message sent to library so that the GAS
+        // is charged
         LOG_GENERAL(WARNING, "Library being called");
-        receipt.AddError(LIBRARY_AS_RECIPIENT);
-        return false;
+        // receipt.AddError(LIBRARY_AS_RECIPIENT);
+        // return false;
```

Comment:
```suggestion
```

Reply:
Clang-format tools takes care of it automatically.